### PR TITLE
Fifth commit

### PR DESCRIPTION
**Pull request by GrigoreCiurea**

### CODE REVIEW

File: src/main/java/org/example/homework_nr_5/Weekday.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package org.example.homework_nr_5;
+
+public enum Weekday {
```
Comment (gh3rtzz) [inline src/main/java/org/example/homework_nr_5/Weekday.java:3]: All values in an enum MUST be capitalised. In month enum all values are capitalised, do the same here

File: src/main/java/org/example/homework_nr_5/Weekday.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    monday, tuesday, wednesday, thursday, friday, saturday, sunday;
+    public  boolean isWeekDay(){
```
Comment (gh3rtzz) [inline src/main/java/org/example/homework_nr_5/Weekday.java:5]: maybe?
`if(this != SATURDAY && this != SUNDAY) return true else return false`

File: src/main/java/org/example/homework_nr_5/Weekday.java (hunk 1 above)
Comment (GrigoreCiurea) [reply src/main/java/org/example/homework_nr_5/Weekday.java:3]: I changed what you request, you can check again?

**Review by gh3rtzz — APPROVED** [review]
approved

**Pull request merged**